action.yml

```diff
@@ -0,0 +1,41 @@
+name: "Foresight Test Kit"
+description: "Test and Coverage ultimate analyzer"
+author: "Thundra"
+inputs:  
+  github_token:
+    description: "GitHub API Access Token"
+    default: ${{ github.token }}
+    required: false
+  api_key: 
+    description: "Foresight Api Key"
+    required: true
+  test_framework:
+    description: "Foresight Test Framework"
+    default: ''
+    required: false
+  test_path:
+    description: "Foresight Test Results Path"
+    default: ''
+    required: false
+  coverage_framework:
+    description: "Foresight Coverage Framework"
+    default: ''
+    required: false
+  coverage_path:
```

Comment:
`coverage_format` might be better name because in fact we are integrating with coverage file formats, not the coverage frameworks.
/cc @yusuf-erdem